src/app/[lang]/(protected)/chat/delete/deleteChat.ts

```diff
@@ -0,0 +1,24 @@
+"use server";
+
+import { env } from "@/env.mjs";
+import { getJwtAccessToken } from "../../regenerateAccessToken";
+
+export async function deleteChat(chatRoomUuid: string | null) {
```

Comment:
In this function, we only pass a valid not nullable chat id, so the type of the parameter should be just a string